test/unit-tests/expression/parse.test.js

```diff
@@ -614,6 +614,25 @@ describe('parse', function () {
       assert.deepStrictEqual(parseAndEval('[[[1],[2]],[[3],[4]]]'), math.matrix([[[1], [2]], [[3], [4]]]))
     })
 
+    it('should parse a matrix with trailing commas', function () {
```

Comment:
Can you add some unit tests validating that multiple trailing commas/semicolons are not allowed? 

Like `[1,2,3,,]` and `[1,2;3,4;,]`.